cms/djangoapps/contentstore/utils.py

```diff
@@ -310,3 +310,16 @@ def reverse_usage_url(handler_name, usage_key, kwargs=None):
     Creates the URL for handlers that use usage_keys as URL parameters.
     """
     return reverse_url(handler_name, 'usage_key_string', usage_key, kwargs)
+
+
+def has_active_web_certificate(course):
```

Comment:
It seems to me this is also a good place to perform the "CERTIFICATES_HTML_VIEW" feature flag check, because we seem to perform these two checks as a pair in most places.